packages/grid/_modules_/grid/components/toolbar/GridDensitySelector.tsx

```diff
@@ -99,7 +107,14 @@ export function GridDensitySelector() {
         onClickAway={handleDensitySelectorClose}
         position="bottom-start"
       >
-        <MenuList role="listbox" onKeyDown={handleListKeyDown} autoFocusItem={Boolean(anchorEl)}>
+        <MenuList
+          id={densityMenuId}
+          className="MuiDataGrid-gridMenuList"
+          role="menu"
```

Comment:
⚠️

Reply:
Removed that one and the one in the export selector